### ISSUE TITLE
Wrong glyph for U+08E2 ARABIC DISPUTED END OF AYAH

### ISSUE THREAD

**Issue by dscorbett** [googlei18n/noto-fonts#1148]
### Font 
NotoSansArabic-Regular.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansArabic-unhinted.zip
Date: 2018-02-04

### Font Version
Version 2.000;GOOG;noto-source:20170915:90ef993387c0

### Issue
U+08E2 ARABIC DISPUTED END OF AYAH should look similar to a large U+06F5 EXTENDED ARABIC-INDIC DIGIT FIVE, but in Noto Sans Arabic it looks similar to U+06DD ARABIC END OF AYAH. Any following digits should appear below the glyph of U+08E2, not beside it. [L2/14-095](https://www.unicode.org/L2/L2014/14095-quranic-marks.pdf#page=4) has an example.

### Character data
࣢١٢٣
U+08E2 ARABIC DISPUTED END OF AYAH
U+0661 ARABIC-INDIC DIGIT ONE
U+0662 ARABIC-INDIC DIGIT TWO
U+0663 ARABIC-INDIC DIGIT THREE

### Screenshot
![࣢١٢٣](https://user-images.githubusercontent.com/1124347/38884518-e113aaf6-423d-11e8-8b1e-2755a3914e32.png)

**Comment by nizarsq** [comment]
I agree [U+08E2](https://unicode.org/charts/PDF/U08A0.pdf) ARABIC DISPUTED END OF AYAH should look similar to U+06F5. However, I never saw digits appear below this glyph or found and official document mention that.
<img width="1205" alt="Screen Shot 2020-05-30 at 12 22 58 AM" src="https://user-images.githubusercontent.com/22158882/83322362-db8cd000-a20b-11ea-9e03-b4e3c2b20d8e.png">

**Comment by dscorbett** [comment]
>I never saw digits appear below this glyph

See [L2/14-095 figure 2](https://www.unicode.org/L2/L2014/14095-quranic-marks.pdf#page=4) for an example.

>or found and official document mention that.

[PropList.txt](https://unicode.org/Public/13.0.0/ucd/PropList.txt) assigns Prepended_Concatenation_Mark to U+08E2, indicating that it has special rendering behavior with a following sequence of digits.
